src/DataModel/IResource.ts

```diff
@@ -0,0 +1,5 @@
+export interface IResource
+{
+    "@id": string;
+    "@type"?: Array<string>
```

Comment:
> Also, we'd need to extend every single JSON object from the payload with that method's implementation - it's a drag.

I think this is actually a deeper question. Do we want to use JSON-LD as data representation? Both internally and accessible through Heracles' interfaces? If not, the point above is moot. If yes, this should be made explicit.